### PR TITLE
Update team page

### PR DESCRIPTION
**Pull request by nikblanchet**
* Add people to all roles
* Alphabetize folks and roles section (except for Eric and Conference chair, respectively)
  Spoke with Mikey about this. She thought it was better than trying to determine seniority/value outside the obvious one.
* Add link to Code of Conduct page
* Remove "Filled roles" subsection

Needed for future:
* Add bios for Nik and Jennifer

### CODE REVIEW

File: docs/conf/portland/2018/team.rst [hunk 1]
```diff
@@ -25,67 +45,50 @@ Mikey Ariel
...
-~~~~~~~~~~~~~
+Mo Nishiyama
+~~~~~~~~~~~~
 
-.. include:: /include/bios/sam.rst
+.. include:: /include/bios/mo.rst
 
-Becky Yoose
-~~~~~~~~~~~
+Nikolas Blanchet
```
Comment (thatdocslady) [inline docs/conf/portland/2018/team.rst:53]: You don't have a bio, Nick?

File: docs/conf/portland/2018/team.rst [hunk 2]
```diff
@@ -25,67 +45,50 @@ Mikey Ariel
...
+* **Explore Portland coordinators** - Mo Nishiyama and Alicia Duncan
+* **Hike coordinator** - Eric Holscher
 * **Infrastructure chair** - Samuel Wright
-* **Writing Day coordinator** - Shaun McCance
+* **Job Fair coordinator** - Eric Holscher
 * **Lightning Talk coordinator** - Rose Williams
-* **Communication chair** - Mikey Ariel
-* **Volunteer coordinator** - Becky Yoose
+* **Social media chair** - Mikey Ariel
+* **Speaker wrangler** - Kelly O'Brien
```
Comment (thatdocslady) [inline docs/conf/portland/2018/team.rst:86]: We decided to stop using "wrangler", I guess I missed this in the search/replace - needs to be "coordinator" everywhere

**Review by thatdocslady — COMMENTED** [review]
Amazing! Just a few nits ;)

**Comment by plaindocs** [comment]
Ping @nikblanchet wanna fix this up so I can merge it in?

**Comment by ericholscher** [comment]
Woops. I merged another one with similar changes. But somehow this isn't conflicting. _shrug_

**Comment by ericholscher** [comment]
Merged & fixed this up. Still need a bio from Nik though :)

**Pull request merged**